tests/models/test_bedrock.py

```diff
@@ -1127,11 +1198,13 @@ async def get_user_country() -> str:
         return 'Mexico'
 
     result = await agent.run('What is the largest city in the user country?')
-    assert result.output == snapshot("""\
+    assert result.output == snapshot(
+        """\
 Based on your location in Mexico, the largest city is Mexico City (Ciudad de México). It's not only the capital but also the most populous city in Mexico with a metropolitan area population of over 21 million people, making it one of the largest urban agglomerations in the world.
 
 Mexico City is an important cultural, financial, and political center for the country and has a rich history dating back to the Aztec empire when it was known as Tenochtitlán.\
-""")
+"""
+    )
```

Comment:
These change are a bit weird and I'm surprised our linter isn't complaining about them. Can you please revert them and keep the `"""` on the same line as the `(` or `)`?

After that I'll merge :)